### PR TITLE
Implemented spring controllers, dto and mappers

### PR DESCRIPTION
**Pull request by tarkaiev**

### CODE REVIEW

File: src/main/java/movie/tickets/dao/impl/CinemaHallDaoImpl.java [hunk 1]
```diff
@@ -56,4 +56,15 @@ public CinemaHall add(CinemaHall cinemaHall) {
             throw new DataProcessingException("Can't get all cinema halls", e);
         }
     }
+
+    @Override
+    public CinemaHall get(Long cinemaHallId) {
+        logger.info("Trying to get cinema hall with id " + cinemaHallId);
```
Comment (dariiabutorina) [inline src/main/java/movie/tickets/dao/impl/CinemaHallDaoImpl.java:62]: Do we need logging on SELECT?

File: src/main/java/movie/tickets/model/dto/mapper/MovieDtoMapper.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+import movie.tickets.model.Movie;
+import movie.tickets.model.dto.MovieRequestDto;
+import movie.tickets.model.dto.MovieResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MovieDtoMapper {
+
+    public Movie fromRequestDto(MovieRequestDto dto) {
```
Comment (dariiabutorina) [inline src/main/java/movie/tickets/model/dto/mapper/MovieDtoMapper.java]:
```suggestion
public class MovieDtoMapper {
    public Movie fromRequestDto(MovieRequestDto dto) {
```

**Review by dariiabutorina — COMMENTED** [review]
Good job!

File: src/main/java/movie/tickets/config/AppConfig.java [hunk 3]
```diff
@@ -15,7 +15,8 @@
 @ComponentScan(basePackages = {
         "movie.tickets.dao",
         "movie.tickets.service",
-        "movie.tickets.security"
+        "movie.tickets.security",
+        "movie.tickets.model.dto"
```
Comment (RomanDubovskyi) [inline src/main/java/movie/tickets/config/AppConfig.java]: it's better to locate mappers in service not in model

File: src/main/java/movie/tickets/dao/impl/CinemaHallDaoImpl.java [hunk 4]
```diff
@@ -56,4 +56,15 @@ public CinemaHall add(CinemaHall cinemaHall) {
...
+        try (Session session = sessionFactory.openSession()) {
+            Query<CinemaHall> query
```
Comment (RomanDubovskyi) [inline src/main/java/movie/tickets/dao/impl/CinemaHallDaoImpl.java]: use session.get() instead

File: src/main/java/movie/tickets/dao/impl/MovieDaoImpl.java [hunk 5]
```diff
@@ -55,4 +55,15 @@ public Movie add(Movie movie) {
             throw new DataProcessingException("Can't get all movies", e);
         }
     }
+
+    @Override
+    public Movie get(Long movieId) {
+        logger.info("Trying to get movie with id " + movieId);
+        try (Session session = sessionFactory.openSession()) {
```
Comment (RomanDubovskyi) [inline src/main/java/movie/tickets/dao/impl/MovieDaoImpl.java:62]: here and everywhere

File: src/main/java/movie/tickets/model/dto/MovieSessionResponseDto.java [hunk 6]
```diff
@@ -0,0 +1,31 @@
+package movie.tickets.model.dto;
+
+public class MovieSessionResponseDto {
+    private Long movieSessionId;
+    private String movieTitle;
+    private String movieSessionDate;
+
```
Comment (RomanDubovskyi) [inline src/main/java/movie/tickets/model/dto/MovieSessionResponseDto.java:8]: let's add cinemaHall id as well

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**